packages/plugin-e2e/tests/datasource/configEditor.spec.ts

```diff
@@ -11,6 +11,6 @@ test('valid credentials should return a 200 status code', async ({ createDataSou
   const configPage = await createDataSourceConfigPage({ type: 'grafana-googlesheets-datasource' });
   await page.getByText('Google JWT File', { exact: true }).click();
   await page.getByTestId('Paste JWT button').click();
-  await page.getByTestId('Configuration text area').fill(process.env.GOOGLE_JWT_FILE!);
+  await page.getByTestId('Configuration text area').fill(process.env.GOOGLE_JWT_FILE!.replace(/'/g, ''));
```

Comment:
Just out of curiosity, why do we need to remove the `'` single quotes here?

Reply:
This variable is also used [here](https://github.com/grafana/plugin-tools/blob/query-editor/packages/plugin-e2e/provisioning/datasources/google-sheets-datasource-jwt.yaml#L15) in the provisioning file for google sheets. Seems like the jwt file needs to be wrapped in single quotes for provisioning to work, so that's how the GH secret is defined. That however doesn't work when creating a new sheets ds using the UI, so therefore I'm removing the single quotes.